File: mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/pathtracking/PathTrackingMvpView.java
package com.mifos.mifosxdroid.activity.pathtracking;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.user.UserLocation;

import java.util.List;

/**
 * Created by Rajan Maurya on 24/01/17.
 */

public interface PathTrackingMvpView extends MvpView {

    void showUserInterface();

    void showPathTracking(List<UserLocation> userLocations);

    void showEmptyPathTracking();

    void showError();

}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/pathtracking/PathTrackingMvpView.kt
package com.mifos.mifosxdroid.activity.pathtracking

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.user.UserLocation

/**
 * Created by Rajan Maurya on 24/01/17.
 */
interface PathTrackingMvpView : MvpView {
    fun showUserInterface()
    fun showPathTracking(userLocations: List<UserLocation>)
    fun showEmptyPathTracking()
    fun showError()
}